server/src/tests/integration.rs

```diff
@@ -463,6 +467,30 @@ async fn deposit_to_l2() -> Result<()> {
     Ok(())
 }
 
+async fn deposit_erc20_to_l2() -> Result<()> {
+    let l1_rpc = var("L1_RPC_URL").unwrap();
```

Comment:
The older code was mostly using `expect()` for env vars, but it's a nit.

Reply:
Done in https://github.com/MovedNetwork/op-move/pull/307/commits/5ece865158ea1c6db900db6338c90bc90cfd9c7b